docs/spec/qualifiers.rst

```diff
@@ -272,33 +282,53 @@ details of the syntax:
     SmallInt = Annotated[int, ValueRange(0, 100)]
     SmallInt(1)  # Type error
 
+:pep:`593` and an earlier version of this specification used the term
+"annotations" instead of "metadata" for the extra arguments to
+``Annotated``. The term "annotations" is deprecated to avoid confusion
+with the parameter, return, and variable annotations that are part of
+the Python syntax.
+
+Meaning
+^^^^^^^
+
+The metadata provided by ``Annotated`` can be used for either static
+analysis or at runtime. If a library (or tool) encounters an instance of
+``Annotated[T, x]`` and has no special logic for metadata ``x``, it
+should ignore it and simply treat the type as ``T``. Thus, in general,
```

Comment:
Nit, just to clarify that T may not be precisely a type (i.e. a type expression): 
```suggestion
should ignore it and simply treat the annotation as ``T``. Thus, in general,
```